extensions/sql-database-projects/src/models/project.ts

```diff
@@ -278,6 +278,12 @@ export class Project implements ISqlProject {
 
 		if (result.folders?.length > 0) { // empty array from SqlToolsService is deserialized as null
 			for (var folderPath of result.folders) {
+				// don't add Properties folder since it isn't supported in ADS. In SSDT it isn't a physical folder, but it's specified in legacy sql projects
+				// to display the Properties node in the project tree
+				if (folderPath === constants.Properties) {
```

Comment:
this filtering could happen in DacFx, but in case SSDT ever switches to use the Projects library, filtering here might make more sense since it's a limitation that ADS doesn't support the properties folder